addons/udes_sale_stock/models/sale_order_line.py

```diff
@@ -187,29 +192,22 @@ def _set_ui_is_cancelled(self):
         )
         in_progress_lines = in_progress_moves.mapped("sale_line_id")
         if in_progress_lines:
-            raise ValidationError(
-                _("Cannot cancel order lines with pickings in progress: %s")
-                % (", ".join(in_progress_lines.mapped("name")),)
+            errors.append(
+                f"Cannot cancel order lines with pickings in progress: {', '.join(in_progress_lines.mapped('name'))}"
             )
 
         # Disallow cancellation at certain stages in the outbound process
         # NB: If half of a sale order line has left the warehouse and the other
         #     half has not been picked yet, cancellation should be allowed.
-        uncancellable_moves = moves.filtered(
```

Comment:
Do we need to do this? We defined `moves` above and it doesn't seem to be used anywhere else.